packages/web-components/src/components/quote/__stories__/quote.stories.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { select } from '@storybook/addon-knobs';
+import { html } from 'lit-element';
+import '../quote';
+import '../../link-with-icon/link-with-icon';
+import ArrowRight20 from 'carbon-web-components/es/icons/arrow--right/20.js';
+import textNullable from '../../../../.storybook/knob-text-nullable';
+
+import readme from './README.stories.mdx';
+
+export const Default = ({ parameters }) => {
+  const { copy, quoteMark, sourceHeading, sourceCopy, sourceBottomCopy } = parameters?.props?.Quote ?? {};
+  return html`
+    <dds-quote inverse="false" markType="${quoteMark}">
```

Comment:
Boolean attribute should be "attribute exists or not" thing unless it's `aria-*`. Also, we use dash case for attribute names (whereas we use came case for property names):

```suggestion
    <dds-quote mark-type="${quoteMark}">
```